src/js/modal/exercise-modal.js

```diff
@@ -196,11 +199,15 @@ const exercise = {
   burnedCalories: 220,
   time: 3,
   popularity: 8322,
-  isFavorite: true,
+  isFavorite: false,
 };
 
-export const showModal = () => {
-  openModalExercise(exercise);
+export const showModal = async function () {
```

Comment:
we must change this logic because this is function was made just for testing purporses